### PR TITLE
JSONField in models

### PR DESCRIPTION
**Pull request by Raznak**
Allow to use DictField with dynamic content in models.

### CODE REVIEW

**Comment by nesdis** [comment]
@Raznak Thank you for your contribution. At some point the idea is to have a single `JsonField` instead of a `ListField` and `DictField`. The field will support both, python lists and dict. It would be great if you could implement this.

**Comment by nagjanaonline** [comment]
@Raznak Thanks a lot, you made my day

**Comment by Raznak** [comment]
Bug occurred when executing insert_many sometimes.
Fix it by using insert_one if only one element to save

File: djongo/sql2mongo/query.py [hunk 1]
```diff
@@ -429,12 +429,22 @@ def execute(self):
                 ins[fld] = v
             docs.append(ins)
 
-        res = self.db_ref[self.left_table].insert_many(docs, ordered=False)
+        if len(docs) == 1:
```
Comment (nesdis) [inline djongo/sql2mongo/query.py]: Why exactly is this a bug? pymongo lets you insert one document in insert_many.

File: djongo/models/fields.py [hunk 2]
```diff
@@ -80,6 +80,31 @@ def _client(self):
...
+class JSONField(Field):
+    """
+    MongoDB JsonField (Array or Dictionnary)
+    """
+
+    def __init__(self, *args, **kwargs):
+        self._value = {}
+        super().__init__(*args, **kwargs)
+
+    def __set__(self, instance, value):
```
Comment (nesdis) [inline djongo/models/fields.py]: The Django field implementation does not need to be getter or setter.

File: djongo/sql2mongo/query.py (hunk 1 above)
Comment (Raznak) [reply djongo/sql2mongo/query.py]: I don't recall the exact error, but sometimes the insert fail. This fix ensure that the bug will not occurs again

**Pull request closed without merging**